project_management/site_completeness_checklist.md

```diff
@@ -10,6 +10,7 @@ This list uses [Github Checklists](https://github.com/blog/1375-task-lists-in-gf
 * [ ] Automated Deployments
 * [ ] Preproduction Environments: Disallows indexing
 * [ ] Preproduction Environments: Require Basic Authentication
+* [ ] Preproduction Environments: Allow access without Basic Authentication for users on the Sparkbox network - [instructions] (./project_management/htaccess_snippets.md)

```

Comment:
Could we find a positive written tone in this phrase? Something like _Whitelist Sparkbox network_ or _Allow access to those on the Sparkbox network_